node/lib/vsotask.ts

```diff
@@ -219,6 +219,17 @@ export function getEndpointAuthorization(id: string, optional: boolean): Endpoin
 }
 
 //-----------------------------------------------------
+// Fs Helpers
+//-----------------------------------------------------
+export interface FsStats extends fs.Stats {
+    
+}
+
+export function stats(path: string): FsStats {
+    return fs.statSync(path);
```

Comment:
Add debug writes here.  debug the input.  debug write the properties and isDirectory, file and symbolic link.  Play around with debug accepting object or string and if object, it prints the directory properties if string, number ... ??  Thoughts